tests/test_adaptor_filter.cc

```diff
@@ -25,11 +25,12 @@
  * Boston, MA 02111-1307, USA.
  */
 
+#include <boost/test/unit_test.hpp>
+
 #include "structure_managers/adaptor_filter.hh"
 // TODO(markus): replace the following line after the branch feat/better_tests
 // is merged into master
```

Comment:
I guess this can go away. Somehow I cannot suggest changes directly any more...

Reply:
done!